### DISCUSSION TITLE
矽谷的企業

### DISCUSSION THREAD

**Discussion by Clare0808** [1101-GS4719-C/Python-Community-GS4719-C#2045] [General]
https://www.businessweekly.com.tw/international/blog/3016329

一開始看到文章的標題，會覺得「在那邊工作也太沒保障了吧?」，但仔細看完文章後才發現，那些大公司會成功不是沒有理由的。
這種看似沒有保障的契約關係，實質上會帶給員工相當大的益處，不管是職場上的表現還是心理上的感受，甚至也能使公司本身更加優秀。如果想保住這份工作，最好的方法就是努力的做、積極上進，把最好的一面呈現給上司，藉此也會增加員工們彼此間的競爭力，進而讓公司的表現向上攀升；而如果員工對於這個公司環境感到不滿，隨時也都可以離開，想離開的人心中不會有壓力，而且也不會影響到其他還在職的員工。

各位覺得矽谷的這種工作環境如何呢?歡迎在下方討論!

**Comment by JoannaLai909** [comment]
前段時間我也一直在youtube上看到很多人分享在美國科技公司工作突然被裁員之類的影片，一開始我覺得公司下手很狠，而且讓員工在一夕之間沒了工作，但看了你的觀點，我覺得這樣做未嘗不好，激發員工間的良性競爭，促使員工增進實力，對一間公司而言有益處，對員工本身，他們也能藉由競爭的過程突破自己

**Comment by ds0w0** [comment]
讓勞動市場更彈性自由的同時也減少了薪水小偷的可能，簡直一舉兩得。我認為就算這樣的工作環境不穩定隨時要擔心被裁員，但對於技術的成長和保持專案的品質都有正向影響，也讓員工能夠相互砥礪，體現了去無存菁且追求更好的精神。不過當然每個人都有不同適合的環境，能接受高壓競爭且希望不斷自我成長的人會適合這裡，反之，希望追求穩定生活的人就不適合。

**Comment by hunteryang0108** [comment]
這種企業的做事風格被稱作狼性文化，同事之間表面和諧相處但往往在暗中較勁，這樣帶來的成效就是員工會非常積極的進行產出，互相摩擦出來的火花往往是開創新事物的出發點。但這樣的文化往往也代表著無情，對員工心理層面的負擔非常大，是否要加入這種公司真的要審慎考慮。
